apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -2145,12 +2165,37 @@ static void libc_stdio_memsistream_tc(void)
 
 	struct lib_memsistream_s memsistream;
 
+	/* Check with 0 BUFLEN */
+
+	lib_memsistream((FAR struct lib_memsistream_s *)&memsistream, buf, 0);
+	TC_ASSERT_EQ("lib_memsistream", memsistream.buffer, (FAR char *)(buf));
+
+	memsistream.public.get(&memsistream.public);
+	TC_ASSERT_EQ("memsistream_getc", memsistream.public.nget, 0);
+
+	/* Check with valid BUFLEN */
+
 	lib_memsistream((FAR struct lib_memsistream_s *)&memsistream, buf, STDIO_BUFLEN);
 	TC_ASSERT_EQ("lib_memsistream", memsistream.buffer, (FAR char *)(buf));
 
+	memsistream.public.get(&memsistream.public);
+	TC_ASSERT_EQ("memsistream_getc", memsistream.public.nget, 1);
+
 	lib_memsistream((FAR struct lib_memsistream_s *)&memsistream, buf + 2, STDIO_BUFLEN);
 	TC_ASSERT_EQ("lib_memsistream", (memsistream.buffer - (FAR char *)(buf)), 2);
+
 	TC_ASSERT_EQ("lib_memsistream", memsistream.buflen, STDIO_BUFLEN);
+	memsistream.public.seek(&memsistream.public, SEEK_OFFSET, SEEK_SET);		/* Seek from the start of the file */
+	TC_ASSERT_EQ("memsistream_seek", memsistream.offset, SEEK_OFFSET);
+
+	memsistream.public.seek(&memsistream.public, SEEK_OFFSET, SEEK_CUR);		/* Seek from the current file offset */
+	TC_ASSERT_EQ("memsistream_seek", memsistream.offset, SEEK_OFFSET * 2);
+
+	memsistream.public.seek(&memsistream.public, SEEK_OFFSET, SEEK_END);		/* Seek from the end of the file */
```

Comment:
Although this tc is passed, offset value is not changed because it was failed.
when SEEK_END, new position is caculated as buflen + offset.
In this case, It will be return ERROR because calcuated value is out of valid range.